src/pages/patch/patchTabs/Tasks.tsx

```diff
@@ -18,14 +25,15 @@ export const Tasks: React.FC = () => {
     notifyOnNetworkStatusChange: true
   });
 
-  if (loading) {
+  if (loading || patchLoading) {
     return <div>Loading...</div>;
   }
   if (error) {
     return <div>{error.message}</div>;
   }
   return (
     <>
+      <P2>{data.patchTasks.length}/{patch.taskCount} tasks</P2>
```

Comment:
in the event of an error on the patch query,  `patch.taskCount` will be undefined. i would conditionally render this entire line based on if `taskCount` is truthy.